python/Cargo.toml

```diff
@@ -39,7 +39,7 @@ features = ["extension-module", "abi3", "abi3-py37"]
 [dependencies.deltalake]
 path = "../rust"
 version = "0"
-features = ["azure", "gcs", "python", "datafusion", "unity-experimental"]
+features = ["azure", "gcs", "python", "datafusion", "unity-experimental", "hdfs"]
```

Comment:
Could we make this an optional feature of the Python library, since it has that extra java dependency?

You can add below:

```toml
hdfs = ["deltalake/hdfs"]
```

And then in Python release add it to the feature flags for release:

https://github.com/delta-io/delta-rs/blob/2077e6a6d0c47f74c83c7bc908517a02a6ecefee/.github/workflows/python_release.yml#L14